ledger/sandbox-classic/BUILD.bazel

```diff
@@ -470,11 +457,9 @@ server_conformance_test(
         "--concurrent-test-runs=1",  # sandbox classic doesn't scale well with concurrent tests (almost no effect on overall run time)
         "--timeout-scale-factor=2",  # sandbox classic is slow in general
         "--open-world",
+        # This test case runs multiple concurrent tests, which don't work well with h2
+        "--exclude=CommandDeduplicationIT:DeduplicationMixedClients",
```

Comment:
Is this a known issue? Do we know why?

Reply:
By magic, it now runs without issue.

The issue was that the tests were a lot slower to run, and because we were running them in parallel it was a race condition, sometimes test cases waited during execution for more than the deduplication duration.

Reply:
Afaik there's an H2 upgrade in progress on main, my suspicion is that somehow triggered that behavior. I will keep an eye on it.